pip_review/__main__.py

```diff
@@ -269,57 +163,60 @@ def confirm(question):
         answer = answer.strip().lower()
     return answer == 'y'
 
+def parse_legacy(pip_output):
+    packages = []
+    for line in pip_output.splitlines():
+        package = {}
+        line = line.split(" - ")
+        package['name'] = re.findall(r'^[a-zA-Z0-9\-]+', line[0])[0]
+        package['version'] = re.findall(r'\(([0-9a-zA-Z\.]+)\)', line[0])[0]
+        package['latest_version'] = re.findall(r'(^[0-9a-zA-Z\.]+)', line[1].split(":")[1].strip())[0]
+        packages.append(package)
+    return packages
+
+
+def get_outdated_packages(unknown):
+    command = ['pip', 'list', '--outdated'] + unknown
+    if parse_version(pip.__version__) > parse_version('9.0'):
+        command.append('--disable-pip-version-check')
+        command.append('--format=json')
+        output = check_output(" ".join(command)).decode('utf-8')
```

Comment:
@jgonggrijp This must have slipped in from @bil-elmoussaoui. We *do not* want to do this. The command should be passed as a list.